### PR TITLE
Cop 11698/vulnerability remidiation

### PR DESCRIPTION
**Pull request by hindumathy-cic**
Vulenrabilities remediation for ref-data-api (https://support.cop.homeoffice.gov.uk/browse/COP-11698). This jira link has the explanation for adding the false positives to the whitelist. Also the dev-test evidences are added in the comment section.

### CODE REVIEW

File: whitelist [hunk 1]
```diff
@@ -11,6 +11,4 @@ CVE-2021-27478
 CVE-2021-27482
 CVE-2021-27498
 CVE-2021-27500
-CVE-2021-29940
-GHSA-qgmg-gppg-76g5
```
Comment (aliktb) [inline whitelist:15]: I see this relates to the `validator` package. Is this a dependency in this application or is it a false positive?

File: whitelist [hunk 2]
```diff
@@ -11,6 +11,4 @@ CVE-2021-27478
...
-GHSA-v78c-4p63-2j6c
```
Comment (aliktb) [inline whitelist:16]: Same as above comment regarding `moment-timezone`

**Review by aliktb — COMMENTED** [review]
Looks fine but just a couple of comments on the vulnerabilities

File: whitelist (hunk 1 above)
Comment (hindumathy-cic) [reply whitelist:15]: yes, GHSA-qgmg-gppg-76g5 is from validator package and it is now fixed and removed from the whitelist. It is not a false positive and it needed package upgradation and code change.

File: whitelist (hunk 2 above)
Comment (aliktb) [reply whitelist:16]: ah yep, my bad. I was thinking you added these lines but they've obviously been removed. Apologies

**Review by aliktb — APPROVED** [review]
Looks good to me

**Pull request merged**